asab/zookeeper/builder.py

```diff
@@ -28,14 +28,19 @@
 """
 
 async def build_client(Config, z_url):
+
     # Parse URL
 	url_pieces = urlparse(z_url)
 	url_netloc = url_pieces.netloc
+	url_path = url_pieces.path
 
 	#If there is no location, use implied
 	if not url_netloc:
 		url_netloc = Config["asab:zookeeper"]["servers"]
 
-	#Create and return the client
+	if url_path.startswith("./"):
```

Comment:
also for `if url_path.startswith("../"):` .. which will require a bit more code logic and could be done in a next MR & sprint.